share/spice/syntax/syntax.js

```diff
@@ -0,0 +1,47 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_syntax = function(api_result) {
+        
+        function getParameterByName(name) {
```

Comment:
Actually, we can remove this funciton altogether, we have a `DDG.get_query()` function you should be using